### PR TITLE
Explain TEMP mode in tutorial

### PR DESCRIPTION
**Pull request by david-a-wheeler**

### CODE REVIEW

File: docs/11/index.md [hunk 1]
```diff
@@ -769,27 +769,73 @@ the proof of the syntax (type) claims as well as the essential steps.
...
+You can also export the current state into a URL:
+
+> Click on the
+> icon <img width="16" height="16" src="menu.svg" alt="menu"> (menu)
+> on the top right of the display.
+
+You can then share this URL, to easily share with others your current state.
+Anyone who opens this URL can see this state.
+
+**NOTE**: When you open a URL URL exported this way, Metamath-lamp
```
Comment (expln) [inline docs/11/index.md:807]: `URL URL`

File: docs/11/index.md [hunk 2]
```diff
@@ -769,27 +769,73 @@ the proof of the syntax (type) claims as well as the essential steps.
...
+> You are about to edit in TEMP mode. All changes you do in TEMP
+> mode will be erased upon closing current browser tab. If you want
+> to continue editing in NORMAL mode, please do the following actions:
+> \1) use "Export to JSON" to copy current editor state to the clipboard;
+> \2) open a new tab (or switch to an already opened tab) with
+> metamath-lamp in NORMAL mode; 3) use "Import from JSON" to load the
+> copied editor state from the clipboard.
+
+There's currently no mechanism to switch from temporary (TEMP) mode
+to regular mode.
```
Comment (expln) [inline docs/11/index.md:834]: Do you think it makes sense to change `NORMAL mode` to `regular mode` in the UI too?

**Review by expln — APPROVED** [review]
Yes, everything is correct. One question: do you think it makes sense to change `NORMAL mode` to `regular mode` in the UI too?

File: docs/11/index.md (hunk 2 above)
Comment (david-a-wheeler) [reply docs/11/index.md:834]: I don't understand your question, but my plan is to just say "regular mode" everywhere and not say "NORMAL mode". I don't see any special indicator in the UI for NORMAL/regular mode, but that's okay, you don't want to overwhelm users.

**Comment by david-a-wheeler** [comment]
I'm going to merge this as an improvement. If you think "regular mode" should be something else we can always change it, now that the terminology is at least consistent :-).

File: docs/11/index.md (hunk 2 above)
Comment (expln) [reply docs/11/index.md:834]: @david-a-wheeler 

> my plan is to just say "regular mode" everywhere

I agree. I've just replaced "NORMAL mode" with "regular mode" on the UI too - https://github.com/expln/metamath-lamp/commit/6f24694d4105000c8b779fa15d6870c709ed3cc6

**Pull request merged**